reference/6/Microsoft.PowerShell.Core/Disable-PSRemoting.md

```diff
@@ -1,17 +1,17 @@
 ---
-ms.date:  10/02/2018
+ms.date:  3/22/2019
 schema:  2.0.0
 locale:  en-us
 keywords:  powershell,cmdlet
 online version:  http://go.microsoft.com/fwlink/?LinkId=821472
 external help file:  System.Management.Automation.dll-Help.xml
 title:  Disable-PSRemoting
 ---
-
 # Disable-PSRemoting
 
 ## SYNOPSIS
-Prevents remote users from running commands on the local computer.
+
+Prevents remote users from running commands in PowerShell Core (PSCore) on the local computer.
```

Comment:
We don't distinguish "PowerShell Core" from "PowerShell" or use the abbreviation "PSCore".

The file is version specific so we just call it "PowerShell"